test/test_decomp.py

```diff
@@ -403,6 +410,40 @@ def test_unsupported(t):
     return any(test_unsupported(x) for x in itertools.chain(flat_args, flat_kwargs))
 
 
+core_backward_failures = {
```

Comment:
Some of these take way too long, so I ran pytest with the timeout argument and skipped those.

Reply:
Hi @nkaretnikov, can you please share the timeout setting and command you used? We need to run them on ROCm to see which tests take way too long on ROCm and pptentially add skips for those.

Reply:
@jithunnair-amd I think it was this: `time pytest --timeout=360 -rfxs test/test_decomp.py -k test_quick_core_backward -v 2>&1 | tee decomp.log`. You need to install https://pypi.org/project/pytest-timeout/ for the timeout option, which is why I only ran it locally and not used a decorator. This is not a dependency that PyTorch has.